app/models/offspring.rb

```diff
@@ -3,4 +3,5 @@ class Offspring < ApplicationRecord
   validates :first_name, presence: true, length: { within: 2..60 }
   validates :last_name, presence: true, length: { within: 2..60 }
   validates :user, presence: true
+  has_many :assignments, dependent: :destroy
```

Comment:
Then you would need active. If you want to do that, you would need to store the history of the one assignment, not create several of thems